api/src/main/webapp/WEB-INF/cron_base.yaml

```diff
@@ -80,4 +80,9 @@ cron:
   schedule: every day 21:00
   timezone: America/Chicago
   target: api
+- description: Find and alert users whose initial credits have expired using cloud task
```

Comment:
what do you think about the "run in test-env only at first" idea?

Reply:
That seems good to me. Would that be accomplished by moving this block to cron_test.yaml?